packages/README.md

```diff
@@ -0,0 +1,41 @@
+Packages
+================
+
+A package allows the creation of modular code that cen easily be shared. For example, packages
+can enable use cases to make network requests, integrate with device APIs, and so on. The minimal
+components of a package include:
+
+* **`pubspec.yaml`** A metadata file declaring package name, version, author, etc.
+
+* **`lib`** The lib directory containing the package's public code and minimally contains a single `<package-name>.dart` file.
+
+To implement a pure Dart package, you can either add the functionality inside `lib/<package name>.dart` or add 
+the functionality in several files inside the `lib` directory. 
+
+This directory contains several examples of packages used with Flutter. 
+For more information on how to use or develop new packages, see the [Using Packages](https://flutter.dev/docs/development/packages-and-plugins/using-packages/) guide.
+
+Shared Packages
+===============================
+Flutter supports users sharing their developed packages to the Flutter and Dart community. 
+
+Flutter packages are published to the ['pub.dev'](https://pub.dev/) page. Users can find the most downloaded
+and highly recommended Flutter and Dart packages.
+
+To see additional packages created by other users, see the ['pub.dev'](https://pub.dev/) page.
+
+
+Developing Packages
+===============================
+Flutter is a supportive and creative commnuity and encourages its members to develop new
+packages if no package exists for your specific use case. Please be sure to review the 
```

Comment:
at line 31, `for your specific use case` should be `for their specific use case` since the sentence is talking about the members. Correct me if I am wrong 🙈.

Reply:
It could go either way since I was writing this with the reader in mind, but since the beginning of the sentence was revised it makes more sense to use "their" now